src/ralph/models/converter.py

```diff
@@ -79,44 +108,33 @@ def convert_dict_event(
         ValidationError: When the final converted event is invalid.
     """
 
-    converted_event = {"id": id_}
+    converted_event = {}
     for conversion_item in conversion_set:
-        try:
-            value = conversion_item.get_value(event)
-        except Exception as err:
-            msg = (
-                f"Failed to get the transformed value for field: {conversion_item.src}"
-            )
-            raise ConversionException(msg) from err
+        value = conversion_item.get_value(event, event_str)
```

Comment:
```suggestion
        data = event
        if conversion_item.raw_input:
            data = event_str
        value = conversion_item.get_value(data)
```

Reply:
I would prefer to delegate this check to the `get_value` method. WDYT ?